<launch>
  <arg name="camera_name" default="sony_cam1"/>
  <arg name="camera_id" default="0"/>
  <arg name="device_ip" default="192.168.1.201"/>
  <arg name="camera_calib" default="file://$(find dslr_cam)/config/$(arg camera_name)_info.yaml"/>
  
  <!-- <arg name="svg_file_path" default="4x4_1000-0.svg"/> -->
  <arg name="png_file_path" default="marker_4x4.png"/>


  <node name="$(arg camera_name)"
        pkg="dslr_cam"
        type="fake_camera_node.py"
        output="screen">
    <param name="frame_rate" value="60"/>
    <param name="camera_name" value="$(arg camera_name)"/>
    <param name="camera_frame_id" value="$(arg camera_name)"/>
    <param name="image_width" value="640"/>
    <param name="image_height" value="480"/>
    <param name="camera_info_url" value="$(arg camera_calib)"/>
    <!-- <param name="svg_file_path" value="$(arg svg_file_path)"/> -->
    <param name="png_file_path" value="$(arg png_file_path)"/>

  </node>
</launch>
